Modules/Chatroom/classes/class.ilChatroom.php

```diff
@@ -261,13 +264,15 @@ public static function findDeletablePrivateRooms() : array
 
     public function getDescription() : string
     {
+	// CR: Incompatible types: Expected property of type '\ilObjChatroom|null', 'bool|\ilObject' provided
```

Comment:
This is a `False Positive`, `ilObjectFactory::getInstanceByObjId` will return a PHP type depending on the id's object type.